frame/instrumental-strategy/src/mock/runtime.rs

```diff
@@ -19,7 +19,14 @@ pub type PoolId = u128;
 pub type Moment = composable_traits::time::Timestamp;
 pub type VaultId = u64;
 
-pub const MILLISECS_PER_BLOCK: u64 = 12000;
+const NATIVE_ASSET: CurrencyId = CurrencyId::PICA;
```

Comment:
```
warning: constant is never used: `NATIVE_ASSET`
```

Reply:
[It's from `instrumental` branch ](https://github.com/ComposableFi/composable/blob/f9be10880770ceea4460f44e2ca6ac7ace46139d/frame/instrumental-strategy-pablo/src/mock/runtime.rs#L28)

Reply:
Resolved